app/frontend/components/pages/delegations/delegatePage.js

```diff
@@ -0,0 +1,159 @@
+const {h} = require('preact')
+const connect = require('unistore/preact').connect
+const actions = require('../../../actions')
+const tooltip = require('../../common/tooltip')
+const printAda = require('../../../helpers/printAda')
+const {AdaIcon} = require('../../common/svg')
+
+const formatStakePoolInfo = (className, message) => {
+  return h('div', {class: `stake-pool-info ${className}`}, message)
+}
+
+const CalculatingFee = () => h('div', {class: 'validation-message send'}, 'Calculating fee...')
+
+const getStakePoolValidationMessage = (stakePool, stakePools) => {
+  let className, message
+
+  const addressUsages = stakePools.filter((pool) => pool.address === stakePool.address)
+  const isDuplicate = addressUsages.length > 1 && stakePool.ID !== addressUsages[0].ID
```

Comment:
I am confused. What are we trying to validate here? Importantly, why are we validating against only [0] ?

Reply:
If the user has typed a stake pool multiple times, I would like to display a warning message to the duplicates (that is, except the first occurrence, with index `[0]`)